redbot/cogs/mod/kickban.py

```diff
@@ -246,7 +279,7 @@ async def kick(self, ctx: commands.Context, user: discord.Member, *, reason: str
     async def ban(
```

Comment:
As proposed earlier in Discord by Flame, it would be good if `ban`'s help docstring said that you need to use user ID/mention if the user isn't in the server.